PrivateCloud.DiagnosticInfo/PrivateCloud.DiagnosticInfo.psm1

```diff
@@ -1355,7 +1364,19 @@ function Get-SddcDiagnosticInfo
 
         [parameter(ParameterSetName="WriteC", Mandatory=$false)]
         [parameter(ParameterSetName="WriteN", Mandatory=$false)]
-        [string] $SessionConfigurationName = $null
+        [string] $SessionConfigurationName = $null,
+
+        [parameter(ParameterSetName="WriteC", Mandatory=$false)]
+        [parameter(ParameterSetName="WriteN", Mandatory=$false)]
+        [string] $DestPath = $($env:userprofile + "\HealthTest"),
+
+        [parameter(ParameterSetName="WriteC", Mandatory=$false)]
+        [parameter(ParameterSetName="WriteN", Mandatory=$false)]
+        [bool] $ZipFiles = $true,
+
+        [parameter(ParameterSetName="WriteC", Mandatory=$false)]
+        [parameter(ParameterSetName="WriteN", Mandatory=$false)]
+        [switch] $ExcludeLocaleMetadata = $false

```

Comment:
changed to bool (used this parameter in gather section per comment to not collect metadata instead of deleting after the fact. Decided to use bool instead of switch)